src/components/color-picker/examples/color-picker.tsx

```diff
@@ -0,0 +1,20 @@
+import { Component, h } from '@stencil/core';
+
+/**
+ * Basic example
+ */
+@Component({
+    tag: 'limel-example-color-picker',
+    shadow: true,
+})
+export class ColorPickerExample {
+    public render() {
+        return (
+            <limel-color-picker
+                tooltipLabel="Click to pick a color"
+                helperText="You can also type a color name or value to preview it here"
+                label="Chosen color"
+            />
```

Comment:
We should listen for the `change` event in the example and show the user the selected value somehow